### PR TITLE
Fixed merge conflict, added student entry etc.

### PR DESCRIPTION
**Pull request by ryushar**
# Requirements

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

- Fixed the merge conflict as mentioned in #136 
- Added back Jekyll #136 
- Added own entry in students section (task)
- Did some improvements and fixes in design

Live link: https://2coolife.github.io/gci17.fossasia.org/

### CODE REVIEW

**Comment by nzec** [comment]
Merge the commits please!

**Comment by isuruAb** [comment]
@2coolife could you please squash all these commits.
please follow this video tutorial to learn about squashing
https://www.youtube.com/watch?v=vp7-NeURmFg&t=35s

**Comment by abishekvashok** [comment]
@2coolife  pls properly rebase your changes. I think you have made commits instead of a single merge commit.

**Comment by ryushar** [comment]
ok now

File: _data/mentors.yml [hunk 1]
```diff
@@ -86,8 +92,12 @@
 - name: Anshuman Verma
   image: ansh.jpg
   twitter: Anshumaniac12
+ 
+- name: Rahul Tyagi
```
Comment (ba11b0y) [inline _data/mentors.yml]: My name is Rahul Tiwari and twitter handle is invinciblycool
Please make the changes

**Review by ba11b0y — CHANGES_REQUESTED** [review]

**Comment by ba11b0y** [comment]
@2coolife Actually I had overlooked some recent commits and merged it, which was the cause of the issue.
Thanks very much for resolving the issue. Please make the requested changes and wait for review.

**Comment by ryushar** [comment]
Did the changes but mentors.yml is getting a merge conflict now. : /

**Review by anshumanv — APPROVED** [review]

**Comment by anshumanv** [comment]
Thanks @2coolife :smile:

**Pull request merged**